blueprints/ember-redux-shim/index.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
         return this.addAddonsToProject({
           packages: [
             {name: 'ember-lodash-shim', target: '^2.0.0'},
-            {name: 'ember-symbol-observable', target: '0.1.2'}
+            {name: 'ember-symbol-observable', target: '0.1.2'},
+            {name: 'babel-plugin-transform-object-rest-spread', target: '6.23.0'}
```

Comment:
I think it is in both places right now - you saying that I can remove it here and "ember install" will drop this into the node_modules anyway? Wasn't sure that was the status quo with shim like addons